src/holdingpen/components/CategoryCard.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Card, List } from 'antd';
+import PropTypes from 'prop-types';
+
+class CategoryCard extends Component {
+  render() {
+    const { title, collectionsCount } = this.props;
+    return (
+      <Card title={title} style={{ width: 300 }}>
+        <List
+          bordered
+          dataSource={Object.keys(collectionsCount)}
+          renderItem={item => (
+            <List.Item>
+              {item} - {collectionsCount[item]}
+            </List.Item>
+          )}
+        />
+      </Card>
+    );
+  }
+}
+
+CategoryCard.propTypes = {
+  title: PropTypes.string,
+  collectionsCount: PropTypes.objectOf(PropTypes.string)
```

Comment:
Not sure if this is the correct way of validating the `collectionsCount` object. The object will have dynamic key names and the value of each would be an Integer, like so:
`{
   'HEP': 10,
   'JOBS': 20,
   ...
}`
Should I use a custom validator?